From linux@blala.de  Tue Jun  1 13:24:56 2010
From: Alexander Kurz <linux@blala.de>
Date: Thu, 20 May 2010 00:32:42 +0400 (MSD)
Subject: Staging: comedi: fixing ni_tio to mite PCI dependancy
To: Randy Dunlap <randy.dunlap@oracle.com>
Cc: Greg KH <gregkh@suse.de>, linux-next@vger.kernel.org, fmhess@users.sourceforge.net, Greg KH <greg@kroah.com>, Alexander Kurz <linux@blala.de>, Ian Abbott <abbotti@mev.co.uk>, Dominik Brodowski <linux@dominikbrodowski.net>, devel@driverdev.osuosl.org, linux-kernel@vger.kernel.org
Message-ID: <alpine.DEB.1.10.1005200028420.20214@blala.de>


On Wed, 19 May 2010, Randy Dunlap wrote:
> linux-next of 2010-0519:
> when CONFIG_PCI is not enabled:
> 
> drivers/staging/comedi/drivers/mite.c: In function 'mite_init':
> drivers/staging/comedi/drivers/mite.c:89: error: implicit declaration of function 'pci_dev_get'
> drivers/staging/comedi/drivers/mite.c:89: warning: assignment makes pointer from integer without a cast
> make[5]: *** [drivers/staging/comedi/drivers/mite.o] Error 1

This patch fixes the problem.

Signed-off-by: Alexander Kurz <linux@kbdbabel.org>
Acked-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/comedi/Kconfig
+++ b/drivers/staging/comedi/Kconfig
@@ -1267,8 +1267,8 @@ config COMEDI_MITE
 
 config COMEDI_NI_TIO
 	tristate "NI general purpose counter support"
+	depends on COMEDI_MITE
 	select COMEDI_8255
-	select COMEDI_MITE
 	default N
 	---help---
 	  Enable support for National Instruments general purpose counters.
